src/SecuredLinksControlTrait.php

```diff
@@ -41,7 +41,7 @@ public function signalReceived(string $signal): void
 
 		if (method_exists($this, $method)) {
 			$reflection = new \ReflectionMethod($this, $method);
-			$secured = Nette\Application\UI\ComponentReflection::parseAnnotation($reflection, 'secured') !== NULL;
+			$secured = Nette\Application\UI\ComponentReflection::parseAnnotation($reflection, 'secured') !== NULL || count($reflection->getAttributes(Secured::class)) > 0;
```

Comment:
This needs to be iffed / or possibly, just remove support for PHP<8.0.

Reply:
Weird that phpstan is ok with this.

Reply:
Dropping PHP<8.0 should be your call in this project :-)
I will add the iffs. What confused me is that tests passed for PHP<8, but that's because tests currently don't cover this part of code.

Reply:
@hrach Added method_exists.
Also added test coverage for signalReceived, wasn't covered before.
Only covers `@secured`, not `#[Secured]` at the moment.